### PR TITLE
Add support for new Middle East Region in location constants

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/nodejs/region.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export let USEast1Region: Region      = "us-east-1";
 export let USEast2Region: Region      = "us-east-2";
 export let USWest1Region: Region      = "us-west-1";
 export let USWest2Region: Region      = "us-west-2";
+export let MESouth1: Region           = "me-south-1";
```
Comment (mikhailshilkov) [inline sdk/nodejs/region.ts]: Consider keeping the alphabetic order of the regions

Comment (stack72) [reply sdk/nodejs/region.ts]: Done!

**Comment by mikhailshilkov** [comment]
Do we also miss `me-south-1` `cn-north-1` `cn-northwest-1` `eu-north-1`?

**Comment by stack72** [comment]
> Do we also miss `me-south-1` `cn-north-1` `cn-northwest-1` `eu-north-1`?

This is me-south-1 but I will add the others now! Good find :)

**Pull request merged**